RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -630,7 +630,7 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Wählen Sie einen Parameter und benutzen Sie die Schaltflächen oder doppelklicken Sie um ihn zu bewegen oder zu löschen</value>
   </data>
   <data name="Restore" xml:space="preserve">
-    <value>Wiederherstellen</value>
+    <value>Wieder-herstellen</value>
```

Comment:
Adding in a "hard" `-` in these places is not helpful, especially when the UI can change any time. It's not guaranteed that this will remain the place to break a line. 

As I understood, the **complete** UI is scheduled for an overhaul with XAML. This would make the change proposed here completely superfluous, if not harmful

---

Ein harten `-` hier Einzufügen bringt uns nicht weiter, da die Oberfläche sich jederzeit ändern kann. Dann ist ein Zeilenumbruch womöglich überflüssig.

Soweit ich weiß, sind alle Oberflächen für eine überarbeitung mit XAML vorgesehen. Das mach die Änderung hier überflüssig, wenn nicht schädlich